### ISSUE TITLE
create individual event pages

### ISSUE THREAD

**Issue by shuchibooksmith** [sonnetmedia/brooklinebooksmith.com#121]
so we used to have individual pages for each author event on the aba site. we'd like to have individual author pages on the new site, so that a customer can click on a box on the event page and get more information. this also makes it easier for the authors to share, for us to include direct links to each event in b-mail, etc. this is something that we hope zoe can create for each author once you've created the template, but we want to move away from having individual author pages on the ABA site. 
thoughts?

**Comment by budparr** [comment]
Okay, I'll include space for author photos on the individual pages (not really room on the event listing pages, though I can re-do those too).

**Comment by shuchibooksmith** [comment]
that'd be great! and if there isn't room on te individual event listing, maybe we'll swap out book for author photo. i'll talk with the group, we'll work on the individual listing pages first. thanks!

**Comment by shuchibooksmith** [comment]
hi @budparr! just wanted to check in on this - thanks!

**Comment by budparr** [comment]
Thanks, Just getting caught up now. I'll do this next.

**Comment by budparr** [comment]
I haven't added author photos yet, but here's an individual event page. Event pages are linked to from each event title.
http://www.brooklinebooksmith.com/events/2016-02/lauret-savoy-trace-memory-history-race-and-the-american-landscape/

**Comment by shuchibooksmith** [comment]
okay great! i imagine we'll want to include more info on each individual page. this looks great though. and this is something @zchyde should be able to easily edit/create?

**Comment by budparr** [comment]
Whenever you create an event the page for it will be created.

**Comment by shuchibooksmith** [comment]
excellent. thank you!

**Comment by shuchibooksmith** [comment]
is there a way for the default individual page to always use the book cover as the banner? or do we have to manually change each banner?

**Comment by budparr** [comment]
The book cover wouldn't work. We're getting them from the ABA side and they're usually quite small and the wrong ratio. I thought about using a default banner image for them, but then you risk getting something quite wrong for a particular event.

**Comment by shuchibooksmith** [comment]
ahh okay, i hadn't thought about the image size. and yeah, you're definitely right about a regular default banner image not being right for all the events. this is something we can discuss here later, maybe @zchyde will have some ideas, but for now it isn't a priority. thanks!

**Comment by budparr** [comment]
@shuchibooksmith  - I realized that I had already allowed for author (or any) additional images. I put an image in here: http://www.brooklinebooksmith.com/events/2016-03/theresa-rebeck-im-glad-about-you/ and anything that goes into the "image path" field in the CMS will show up. At one time I had that supersede the book cover image, but on [Jan 13](https://github.com/sonnetmedia/brooklinebooksmith.com/commit/9ef8a75424215e914f5f04817a80de3c4acf6aec#diff-86fa898feedc7c5b349b97995be0e6ce) I made a change to allow them to co-exist. Whether or not this is the best way to handle them is another question, but this style allows for both or either/or.
